### PR TITLE
Cleanup php.ini

### PR DESCRIPTION
**Pull request by J0WI**
- `php.ini` has been dropped in favor of a `RUN` instruction
- `open_basedir` can be dropped since the application runs in a container anyway
- `upload_max_filesize`/`post_max_size` are removed due https://github.com/docker-library/wordpress/issues/10
- `session.hash_function` has been [removed in PHP 7.1](https://www.php.net/manual/en/session.configuration.php#ini.session.hash-function)
- `session.save_path` can be set manually if needed (there are also use cases with `memcached`/`redis` as session handler, just let the user choose)
- using recommend defaults for `opcache`
- minor code style fixes in Dockerfiles

### CODE REVIEW

**Comment by patschi** [comment]
What about adding `expose_php` as in #209 directly to your commit? Would make sense IMHO.

**Comment by J0WI** [comment]
I think this should be solved by PhpMyAdmin upstream or mitigated by a proxy.

**Comment by patschi** [comment]
How is this related directly to PMA? The header is added by PHP itself, not by the application. Even the fact I've also "solved" (more likely workarounded IMHO) by removing the header using a reverse proxy, not everyone might use a proxy.

**Comment by J0WI** [comment]
I mean there is no upstream recommendation or example configuration (like a `.htaccess` or `.user.ini`) for this. I expect that production container run behind a proxy where custom headers like HSTS can be added and `X-Powered-By` can be removed.
But I don't see a strong reason to remove this header. Security through obscurity won't stop any bots attacking your container.

File: Dockerfile-debian.template [hunk 1]
```diff
@@ -31,46 +31,67 @@ RUN set -ex; \
...
+	} > /usr/local/etc/php/conf.d/phpmyadmin-misc.ini
+
 # Calculate download URL
 ENV VERSION 4.8.5
 ENV URL https://files.phpmyadmin.net/phpMyAdmin/${VERSION}/phpMyAdmin-${VERSION}-all-languages.tar.xz
 LABEL version=$VERSION
 
 # Download tarball, verify it using gpg and extract
 RUN set -ex; \
-    fetchDeps=" \
```
Comment (ibennetch) [inline Dockerfile-debian.template]: I believe your editor has converted many of these spaces to tabs, is that intentional? Throughout phpMyAdmin's repositories, we try to stick with spaces unless there's a compelling reason to use a tab isntead.

Comment (J0WI) [reply Dockerfile-debian.template]: I converted them to spaces now.

**Comment by J0WI** [comment]
@williamdes @ibennetch would you like to review this?

**Comment by williamdes** [comment]
Merged !
Thanks a lot for your help @J0WI !

**Pull request merged**